sim/pkg/simulator/fakes.go

```diff
@@ -121,3 +121,7 @@ func (fd *fakeDispatcher) VerticalRecommendation(partition string, time int64) (
 func (fd *fakeDispatcher) GetCapabilities() (rec []proto.Capability, err error) {
 	return []proto.Capability{}, nil
 }
+
+func (fd *fakeDispatcher) PluginType() (rec string, err error) {
```

Comment:
This is okay for now.  Perhaps add a TODO to combine these into a `Metadata` API, because there will be more we want to return from the plugin, like default configuration etc..

Reply:
Added it